content/en/tracing/trace_collection/library_config/dotnet-framework.md

```diff
@@ -240,19 +220,70 @@ The `from-key` value is specific to the integration type, and should exclude the
 `DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
 : When `DD_HTTP_SERVER_TAG_QUERY_STRING` is true, this regex redacts sensitive data from incoming requests' query string reported in the `http.url` tag (matches are replaced with `<redacted>`). This regex executes for every incoming request.
 
+#### Agent
+
+`DD_AGENT_HOST`
+: Sets the host where the Agent is listening for connections. Can be a hostname or an IP address. Use `DD_TRACE_AGENT_URL`, which has precedence over this parameter. <br>
+**Default**: `localhost`
+
+`DD_TAGS`
```

Comment:
The DD_TAGS environment variable is read both by the SDKs (adds the specified tags to all spans) and by the Datadog Agent. I'm not sure if it makes sense to list this config twice on the same page, so I think the better place to put it in our APM docs is under the `### Traces` section

Reply:
Thanks! Fixed.